pkg/testing/envtest_setup.go

```diff
@@ -38,9 +38,9 @@ func SetupEnvTest() *envtest.Environment {
 		// The relative paths must be provided for each level of test nesting
 		// This code should be illegal
 		CRDDirectoryPaths: []string{
-			filepath.Join("..", "..", "..", "..", "..", "config", "default", "crds", "base"),
+			filepath.Join("..", "..", "..", "..", "..", "config", "crd"),
```

Comment:
Do we still need both of these paths?

Reply:
yes, we have tests at different levels
`pkg/controller/inferenceservice/reconcilers/knative`
`pkg/controller/inferenceservice`